ngraph/frontend/cmake_static_protobuf/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ set(BUILD_SHARED_LIBS OFF)
 set(BUILD_STANDALONE_STATIC ON)
 set(USE_STATIC_PROTOBUF ON)
 
-include(../../cmake/external_protobuf.cmake)
+add_subdirectory(../../../thirdparty/protobuf ${CMAKE_BINARY_DIR}/_deps/static-protobuf)
 
 set(BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS_OLD})
```

Comment:
can we move this whole script to `thirdparty`? while frontends will just reinvoke it

Reply:
I will try to do that.